e2e/interchaintestv8/relayer/solana_and_cosmos_config.go

```diff
@@ -50,6 +52,8 @@ type CosmosToSolanaModuleConfig struct {
 	SolanaIcs26ProgramId string `json:"solana_ics26_program_id"`
 	// Solana ICS07 Tendermint light client program ID (must be "solana_ics07_program_id")
 	SolanaIcs07ProgramId string `json:"solana_ics07_program_id"`
+	// Solana access manager program ID (must be "solana_access_manager_program_id")
+	SolanaAccessManagerProgramId string `json:"solana_access_manager_program_id"`
```

Comment:
These should actually be queried. But I'll create a seperate issue for this https://github.com/cosmos/solidity-ibc-eureka/issues/821